docs/articles/documentation/guides/advanced-guides/authentication.md

```diff
@@ -134,50 +111,38 @@ test('Anonymous users can see newly created comments', async t => {
         .click('#submit')
         .useRole(Role.anonymous());
 
-        const comment = await Selector('#comment-data');
+    const comment = await Selector('#comment-data');
 
-        await t.expect(comment.innerText).eql('Hey ya!');
+    await t.expect(comment.innerText).eql('Hey ya!');
 });
 ```
 
-## Role Options
-
-You can pass the following options to the [Role constructor](#create-and-apply-roles).
-
-### options.preserveUrl
-
-Use this option to control which page is opened after you switch to the role.
-
-By default, TestCafe navigates back to the page that was opened previously to switching to the role.
-Set the `preserveUrl` option to `true` to save the URL to which the browser was redirected after logging in.
-TestCafe will navigate to the saved URL each time after you switch to this role.
+## HTTP Authentication
 
-This option is useful if you store session-related data (like session ID) in the URL.
+TestCafe allows you to test web pages that are protected with HTTP [Basic](https://en.wikipedia.org/wiki/Basic_access_authentication) or [Windows (NTLM)](https://en.wikipedia.org/wiki/Integrated_Windows_Authentication) authentication. Use the [test.httpAuth](../../reference/test-api/test/httpauth.md) method to specify the credentials to be used by an individual test and the [fixture.httpAuth](../../reference/test-api/fixture/httpauth.md) method to specify the credentials for the entire fixture.
```

Comment:
Change the second sentence to:
Use the [test.httpAuth](../../reference/test-api/test/httpauth.md) method to specify individual test credentials and the [fixture.httpAuth](../../reference/test-api/fixture/httpauth.md) method to specify credentials for the entire fixture.